### ISSUE TITLE
AgeRatingContentDescription missing some categories

### ISSUE THREAD

**Issue by vulovicv23** [kamranayub/igdb-dotnet#27]
In AgeRatingContentDescription response there are missing categories.
Right now, this is what we have:
```
public enum AgeRatingContentDescriptionCategory
  {
    PEGI = 1,
    ESRB = 2
  }

```  
  
  But we need to include more:
```
public enum AgeRatingContentDescriptionCategory
  {
    ESRB = 1,
    PEGI = 2,
    CERO = 3,
    USK = 4,
    GRAC = 5,
    CLASS_IND = 6,
    ACB = 7
  }
```

**Comment by vulovicv23** [comment]
Already have a branch and solution for this, so, let me know how we can handle this.

Thanks!

**Comment by kamranayub** [comment]
Closed in #29
